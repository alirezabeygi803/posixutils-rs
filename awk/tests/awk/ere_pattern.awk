/Celine|Smith/ {print $0}