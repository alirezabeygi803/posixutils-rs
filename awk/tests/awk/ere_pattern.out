2 Smith sailor 45
5 Celine clergy 87
